CMSgov/mint-app
# [EASI-3926] Frontend work for Add Collaborator Notification

# EASI-3926


## Changes and Description

- Add `AddedAsCollaboratorMeta` fragment to `GetNotification.ts`
- Update Notification e2e test
- DRYed up `handleMarkAsRead` function in `IndividualNotification.tsx`
- Add typeguard to `_utils.tsx`



## How to test this change

1. Add `SF13` as a collabortor to any plan
2. Login as SF13 and see notification in the notification center


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
